build-system/tasks/runtime-test/helpers.js

```diff
@@ -272,17 +86,31 @@ function refreshKarmaWdCache() {
   exec('node ./node_modules/wd/scripts/build-browser-scripts.js');
 }
 
-/**
- * Returns true if the PR is a large refactor.
- * (Used to skip testing local changes.)
- * @return {boolean}
- */
-function isLargeRefactor() {
-  const filesChanged = gitDiffNameOnlyMaster();
-  return filesChanged.length >= LARGE_REFACTOR_THRESHOLD;
-}
-
 function getBrowserConfig() {
+  if (argv.saucelabs) {
+    if (IS_GULP_UNIT) {
+      return {browsers: ['SL_Safari_12', 'SL_Firefox']};
+    }
+
+    if (IS_GULP_INTEGRATION) {
+      return {
+        browsers: [
+          'SL_Chrome',
+          'SL_Firefox',
+          'SL_Edge_17',
+          'SL_Safari_12',
+          'SL_IE_11',
+          // TODO(amp-infra): Evaluate and add more platforms here.
+          //'SL_Chrome_Android_7',
+          //'SL_iOS_11',
+          //'SL_iOS_12',
+          'SL_Chrome_Beta',
+          'SL_Firefox_Beta',
+        ],
+      };
+    }
+  }
```

Comment:
Maybe add an "else throw error" in case something breaks along the way and this part is reached without actually getting here without IS_GULP_UNIT/INTEGRATION?

Reply:
Done!